WordPress/Classes/ViewRelated/Reader/ReaderMenuViewModel.swift

```diff
@@ -234,53 +260,42 @@ enum ReaderDefaultMenuItemOrder: Int {
                     continue
                 }
 
-                var item = ReaderMenuItem(title: abstractTopic.title, type: .topic, icon: nil, topic: abstractTopic)
-                if ReaderHelpers.topicIsFollowing(abstractTopic) {
-                    item.order = ReaderDefaultMenuItemOrder.followed.rawValue
-                    item.icon = Gridicon.iconOfType(.checkmarkCircle)
-                } else if ReaderHelpers.topicIsDiscover(abstractTopic) {
-                    item.order = ReaderDefaultMenuItemOrder.discover.rawValue
-                    item.icon = Gridicon.iconOfType(.mySites)
-                } else if ReaderHelpers.topicIsLiked(abstractTopic) {
-                    item.order = ReaderDefaultMenuItemOrder.likes.rawValue
-                    item.icon = Gridicon.iconOfType(.star)
-                } else {
-                    item.order = ReaderDefaultMenuItemOrder.other.rawValue
-                }
+                let item = sectionCreator(for: abstractTopic).menuItem(with: abstractTopic)
+
                 defaultSectionItems.append(item)
             }
         }
 
-        // Create a menu item for search
-        var searchItem = searchMenuItem()
-        searchItem.order = ReaderDefaultMenuItemOrder.search.rawValue
-        searchItem.icon = Gridicon.iconOfType(.search)
-        defaultSectionItems.append(searchItem)
-
-        // Sort the items into the desired order.
-        defaultSectionItems.sort { (menuItem1, menuItem2) -> Bool in
-            if menuItem1.order < menuItem2.order {
-                return true
-            }
-
-            if menuItem1.order > menuItem2.order {
-                return false
-            }
+        // Append a menu item for search
+        defaultSectionItems.append(searchMenuItem())
 
-            if menuItem1.title < menuItem2.title {
-                return true
-            }
+        // To be removed as soon as the topic is provided by the coredata store. This is here just to prove visually that the view model can handle this topic
+        if FeatureFlag.saveForLater.enabled {
+            let topic = ReaderSaveForLaterTopic()
+            topic.title = "Bookmarked - mock"
```

Comment:
Based on the first iteration of designs we now have (even though this will be removed soon), let's rename this to "Saved Posts" for now.